test/travis.sh

```diff
@@ -56,6 +56,7 @@ if [ "$RULESETS_CHANGED" ]; then
     docker_build
     docker run --rm -ti -v $(pwd):/opt httpse python utils/trivial-validate.py
     docker run --rm -ti -v $(pwd):/opt httpse python utils/ruleset_filenames_validate.py
+    docker run --rm -ti -v $(pwd):/opt httpse bash -c "utils/validate.sh"
```

Comment:
`utils/validate.sh` runs `trivial-validate.py` which is also run two lines above, so there is some redundancy here.